include/seqan3/range/decorator/gap_decorator.hpp

```diff
@@ -321,7 +324,7 @@ class gap_decorator
         return iterator{*this, pos1};
     }
 
-    /*!\brief Assigns a new sequence of type seqan3::gap_decorator::unaligned_seq_type to the decorator.
+    /*!\brief Assigns a new sequence of type seqan3::gap_decorator::unaligned_sequence_type to the decorator.
      * \param[in,out] dec       The decorator to modify.
      * \param[in]     unaligned The unaligned sequence to assign.
      * \details
```

Comment:
A few lines below there is
`template <typename unaligned_seq_t>`

since this is public, can you change it to `template <typename unaligned_sequence_t>` to reflect the general naming scheme?

Can you check if there are other occurrences in the gap_decorator?